packages/babel-plugin-transform-decorators-2/test/fixtures/examples/logger/exec.js

```diff
@@ -0,0 +1,31 @@
+const calls = [];
+
+function log(message) {
+  return function (descriptor) {
+    let oldFunc = descriptor.value;
+    descriptor.value = function(...args) {
```

Comment:
Somewhat unfortunately, there is no way of making this function's name derive from the original function name (instead of "value") other than by calling `defineProperty` on it afterwards 🤔